pkgs/development/tools/rgp/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchurl
+, autoPatchelfHook
+, dbus
+, fontconfig
+, freetype
+, glib
+, libglvnd
+, libX11
+, libxcb
+, libXi
+, ncurses5
+, qtbase
+, zlib
+}:
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "rgp";
+  version = "1.7.0.29";
+
+  src = fetchurl {
+    url = "https://github.com/GPUOpen-Tools/radeon_gpu_profiler/releases/download/v${lib.versions.majorMinor version}/RadeonGPUProfiler_${version}.tgz";
+    sha256 = "d7d4aa92231796a509dfcf92e9618cac98ae6572c12efd0871eb43afa4e8240a";
+  };
+
+  nativeBuildInputs = [ makeWrapper autoPatchelfHook ];
+
+  buildInputs = with pkgs; [
```

Comment:
Sorry, should've caught that in my first review. `with pkgs` is unnecessary here, and `pkgs` isn't actually defined anywhere. This should raise an error, but surprisingly it doesn't.

Reply:
In case you're curious, here is some discussion about why this didn't cause an error: https://logs.nix.samueldr.com/nixos-dev/2020-04-01#3255268;

Reply:
Interesting